docs/index.md

```diff
@@ -0,0 +1,44 @@
+---
+layout: layouts/base.njk
+title: Home
+---
+
+<div class="app-masthead">
+
+  <div class="govuk-width-container">
+
+    <h1 class="govuk-heading-xl app-masthead__title">Design your service using Ministry of Justice styles, components and patterns</h1>
+
+    <p class="app-masthead__description">Use this design system to make your service consistent with Ministry of Justice. Learn from the research and experience of other service teams and avoid repeating work that’s already been done.</p>
+
+    <a class="app-masthead__button govuk-!-margin-top-6 govuk-!-margin-bottom-0" href="/get-started">Get started <svg class="app-masthead__button__icon" xmlns="http://www.w3.org/2000/svg" width="17.5" height="19" viewBox="0 0 33 40" role="presentation" focusable="false">
+      <path fill="currentColor" d="M0 0h13l20 20-20 20H0l20-20z"></path>
+    </svg></a>
+
+  </div>
+
+</div>
+
+<div class="govuk-width-container">
+
+  <div class="app-main-wrapper govuk-main-wrapper govuk-main-wrapper--l">
+
+    <div class="govuk-grid-row">
+
+      <div class="govuk-grid-column-one-third govuk-!-margin-bottom-8">
+        <h2 class="govuk-heading-l">Components</h2>
+        <p>Save time with reusable, accessible components for forms, navigation, panels, tables and more.</p>
+        <p><a class="app-link govuk-!-font-weight-bold" href="/components">Browse components</a></p>
+      </div>
+
+      <div class="govuk-grid-column-one-third govuk-!-margin-bottom-8">
+        <h2 class="govuk-heading-l">Patterns</h2>
+        <p>Help users complete common tasks like entering names and addresses, filling in forms and creating accounts.</p>
+        <p><a class="app-link govuk-!-font-weight-bold" href="/patterns">Browse patterns</a></p>
```

Comment:
All links on this page are absolute instead of relative giving 404s

Reply:
Thanks @johnnolan, this led to me finding a bunch more 404s which I've also now fixed 🎉